### PR TITLE
Improve emails templates

### PR DESCRIPTION
**Pull request by kulczy**
| Q               | A
| --------------- | -----
| Branch?         | master
| Bug fix?        | no
| New feature?    | no
| BC breaks?      | no
| Deprecations?   | no
| License         | MIT

## Shipment confirmation
Zrzut ekranu 2019-11-29 o 13 23 01

## Contact request
Zrzut ekranu 2019-11-29 o 13 24 54

## User registration
Zrzut ekranu 2019-11-29 o 13 23 59

## Email verification
Zrzut ekranu 2019-11-29 o 13 23 55

## Password reset
Zrzut ekranu 2019-11-29 o 13 23 52

## Order confirmation
Zrzut ekranu 2019-11-29 o 13 23 43

### CODE REVIEW

File: src/Sylius/Bundle/AdminBundle/Resources/views/Email/shipmentConfirmation.html.twig [hunk 1]
```diff
@@ -3,11 +3,24 @@
...
-        <br/><br/>
-        Thank you for a successful transaction.
-    {% endautoescape %}
+    {% include '@SyliusAdmin/Email/_layout.html.twig' with {'content': block('content')} %}
+{% endblock %}
+
+{% block content %}
+    <div style="text-align: center;">
+        <div>
+            Your order with number
```
Comment (lchrusciel) [inline src/Sylius/Bundle/AdminBundle/Resources/views/Email/shipmentConfirmation.html.twig]: Can we introduce translations keys (without any further adjustments) to these templates? Or should it be a part of next pr?

Comment (CoderMaggie) [reply src/Sylius/Bundle/AdminBundle/Resources/views/Email/shipmentConfirmation.html.twig]: It is another task in the backlog

**Comment by pjedrzejewski** [comment]
I do not know about Gulp, but with BootstrapTheme and Webpack/Encore, I had an issue with displaying assets (logo, to be specific) in Gmail - it has something to do with image caching done by Gmail and the custom extension (app.XXX.js) added by Webpack.

**Comment by pamil** [comment]
Thanks, Szymon! :tada:

File: src/Sylius/Bundle/ShopBundle/Resources/views/Email/userRegistration.html.twig [hunk 2]
```diff
@@ -1,13 +1,18 @@
...
-        Thank you {{ user.username }}.
-    {% endautoescape %}
+{% block content %}
+    <div style="text-align: center; margin-bottom: 30px;">
+        <div style="font-size: 24px;">Welcome to our store!<br></div>
+        You have just been registered. Thank you {{ user.username }}.
+    </div>
+
+    <div style="text-align: center;">
+        <a href="{{ absolute_url(path('sylius_shop_homepage')) }}" style="display: inline-block; text-align: center; background: #1abb9c; padding: 18px 28px; color: #fff; text-decoration: none; border-radius: 3px;">
```
Comment (lchrusciel) [inline src/Sylius/Bundle/ShopBundle/Resources/views/Email/userRegistration.html.twig]: 🚨We missed support for different domains. 🚨 

Maybe the same case for other templates as well

**Pull request merged**